data-raw/00_sourmash_commands.sh

```diff
@@ -0,0 +1,94 @@
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR594/006/SRR5947006/SRR5947006_2.fastq.gz
+
+# SRR5935765
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR593/005/SRR5935765/SRR5935765_1.fastq.gz
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR593/005/SRR5935765/SRR5935765_2.fastq.gz
+
+# SRR5936197
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR593/007/SRR5936197/SRR5936197_1.fastq.gz
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR593/007/SRR5936197/SRR5936197_2.fastq.gz
+
+# SRR5946923
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR594/003/SRR5946923/SRR5946923_1.fastq.gz
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR594/003/SRR5946923/SRR5946923_2.fastq.gz
+
+# SRR5946920
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR594/000/SRR5946920/SRR5946920_1.fastq.gz
+wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR594/000/SRR5946920/SRR5946920_2.fastq.gz
+
+
+###################################################################################
+## SKETCH THE DATA
+###################################################################################
+
+for infile in *_1.fastq.gz
+do
+    bn=$(basename ${infile} _1.fastq.gz)
+    sourmash sketch dna -p k=21,k=31,k=51,scaled=1000,abund --merge ${bn} -o ${bn}.sig ${infile} ${bn}_2.fastq.gz
+done
+
+###################################################################################
+## RUN SOURMASH GATHER
+###################################################################################
+
+wget -O gtdb-rs207.genomic-reps.dna.k31.zip https://osf.io/3a6gn/download
+for infile in *sig
+do
+    bn=$(basename $infile .sig)
+    sourmash gather ${infile} gtdb-rs207.genomic-reps.dna.k31.zip -o ${bn}_gather_gtdbrs207_reps.csv
+done
+
+###################################################################################
+## RUN SOURMASH TAXONOMY
+###################################################################################
+
+wget -O gtdb-rs207.taxonomy.csv.gz https://osf.io/v3zmg/download
+gunzip gtdb-rs207.taxonomy.csv.gz
+sourmash tax prepare -t gtdb-rs207.taxonomy.csv -o gtdb-rs207.taxonomy.sqldb -F sql
+
+for infile in *_gather_gtdbrs207_reps.csv
+do
+    sourmash tax annotate -g ${infile} -t gtdb-rs207.taxonomy.sqldb 
+done
+
+###################################################################################
+## DOWNSAMPLE THE SIGNATURES TO MAKE THEM SMALLER
+###################################################################################
+
+for infile in *sig
+do
+    sourmash sig downsample --scaled 200000 -o ${infile} ${infile}
+done
+
+
+###################################################################################
+## COMPRESS CSV FILES 
+###################################################################################
+
+for infile in *csv
+do
+   gzip ${infile}
+done
+
+####################################################################################
+## RUN SOURMASH COMPARE
+####################################################################################
+
+sourmash compare -k 31 -o gut_compare.csv *sig
```

Comment:
Does this file just stay in `data-raw` to document how the example data was generated or does it become part of the documentation/tutorial vignettes? I know vignettes are usually just focused on the R package docs but would be good to reference this somewhere in the official docs as well

Reply:
it just stays in `data-raw`. I've added a reference to it in the vignette! (the vignette itself is still WIP)